inc/template-tags.php

```diff
@@ -71,12 +97,64 @@ function siteorigin_unwind_entry_footer() {
 			esc_html__( 'Edit %s', 'siteorigin_unwind' ),
 			the_title( '<span class="screen-reader-text">"', '"</span>', false )
 		),
-		'<span class="edit-link">',
+		'</br><span class="edit-link">',
 		'</span>'
 	);
 }
 endif;
 
+if ( ! function_exists( 'siteorigin_unwind_author_box' ) ) :
+/**
+ * Displays the author box in single posts
+ */
+function siteorigin_unwind_author_box() {
+	if ( siteorigin_setting('blog_display_author_box') ) { ?>
```

Comment:
I think we'd want the conditional to be outside the function. Calling `siteorigin_unwind_author_box` should always display the author box, regardless of the `blog_display_author_box` setting. Instead, the conditional should be handled in the template file that calls `siteorigin_unwind_author_box`.

The reason for this is simply to make Unwind more child theme friendly. Lets say a child theme wants to display an author box in a very unusual place. They might disable `blog_display_author_box` in the theme settings to remove it from the default position, then call `siteorigin_unwind_author_box` in one of the child theme files.

The other reason is just to keep our method of handling theme settings consistent. For example, how we're handling it here: https://github.com/siteorigin/siteorigin-unwind/pull/5/files#diff-6ce624532346493b7a0d1a63ecfc68ceR25 and here: https://github.com/siteorigin/siteorigin-unwind/pull/5/files#diff-6ce624532346493b7a0d1a63ecfc68ceR31